File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -447,6 +521,85 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Find-and-Replace/Find-and-highlight-in-PowerPoint).
 
+## Find and highlight using Regex pattern 
+
+You can find all occurrences of a pattern of text in a PowerPoint presentation using the FindAll method and  highlight the found pattern of text.
+
+The following code example illustrates how to find all occurrences of a pattern of text and highlight the found pattern of text in a PowerPoint presentation using Regex pattern.
```

Comment:
@Venkateshmuruganandam - Please refer the below UG topic and modify the statement similarly,
https://help.syncfusion.com/file-formats/presentation/working-with-find-and-replace#find-and-highlight-in-powerpoint-presentation